test/functional/p2p_blocksonly.py

```diff
@@ -57,29 +56,35 @@ def run_test(self):
             self.nodes[0].p2p.wait_for_tx(txid)
             assert_equal(self.nodes[0].getmempoolinfo()['size'], 1)
 
-        self.log.info('Check that txs from forcerelay peers are not rejected and relayed to others')
-        self.log.info("Restarting node 0 with forcerelay permission and blocksonly")
-        self.restart_node(0, ["-persistmempool=0", "-whitelist=127.0.0.1", "-whitelistforcerelay", "-blocksonly"])
+        self.log.info('Check that txs from RELAY-whitelisted peers are not rejected and relayed to others')
```

Comment:
```suggestion
        self.log.info('Check that txs from peers with relay-permission are not rejected and relayed to others')
```

Would prefer to use lowercase (identical to the flag name for the whitelist setting)

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/19963/commits/ee11320b0616ae6811a0aef6a905e62e205a144f

Reply:
I agree with Marco even though it was my suggestion to use the uppercase.
We definitely use lowercase for permissions, I was just referring to the enum, which has less prio in this case.